src_kolko_krzyżyk/kolko.py

```diff
@@ -0,0 +1,76 @@
+class TicTacToe:
+    #class veribles
+    used_nums = ['','','','','','','','','']
+    board = [['0', '1', '2'],
+            ['3', '4', '5'],
+            ['6', '7', '8']]
+    way_to_win = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6],
+                  [1, 4, 7], [2, 5, 8], [0, 4, 8], [6, 4, 2]]
+
+    def __init__(self):
+        self.used_nums = self.used_nums
+        self.board = self.board
+        self.way_to_win = self.way_to_win
+
+    def printer(self):
+        'print board with positions to choose'
```

Comment:
You are using two types of docstrings:
```python
def something():
    'the first one'

def something2():
    '''the second one'''
```

Choose one of them and use it constantly. It doesn't looks good if you mix two of them.